src/FxCop/System.Globalization.Analyzers/Test/Globalization/CA1305SpecifyIFormatProviderTests.cs

```diff
@@ -22,28 +20,54 @@ sealed class C
     void M(string strA, string strB)
     {
         string str1 = string.Format(""Foo {0}"", strA);
-        string str2 = string.Format(""Foo {0} {1}"", strA, strB);
+        string str2 = string.Format(""Foo {0} {1}"", strA, strB); 
+    }
+}",
+            GetCA1305CSharpDefaultResultAt(9, 23, MessageAlternateString,
```

Comment:
That's weird, These tests shouldn't be working. Did these test cases pass when you run them?